components/VoteTicker/calculateTimeRemaining.ts

```diff
@@ -0,0 +1,14 @@
+import { DateTime, Duration } from "luxon";
+
+// For Timer feature.
+// Countdown from UTC 00:00:00.
+export default function calculateTimeRemaining() {
```

Comment:
this should be named something like, timeToEndOfDay, and imo id separate the time in ms and the formatting of that time for display into separate functions

Reply:
I think this function may just be poorly named. In reality it really is just formatting the "UTC time" as we never actually need a time input due to votes always starting at UTC midnight. General case for this is pointless as we should never need a time in MS input.

WDYT?

Reply:
Renamed to be much clearer.